app/views/articles/show.html.erb

```diff
@@ -146,7 +146,16 @@
             </span>
           <% end %>
           <span class="published-at">・<%= @article.reading_time < 1 ? 1 : @article.reading_time %> min read</span>
-          <span class="action-space" id="action-space"></span>
+          <span class="action-space" id="action-space">
+            <% if @article.user == current_user %>
+              <%= link_to "EDIT", edit_user_article_path(username: @article.username, slug: @article.slug) %>
+              <%= link_to "MANAGE", manage_user_article_path(username: @article.username, slug: @article.slug) if @article.published? %>
+            <% elsif current_user&.trusted %>
```

Comment:
This is actually not possible here. We don't have access to `current_user` in this file because it will be cached at the edge.

I actually feel like it would make sense to have some custom linting that would catch this because it's not typical for Rails. This is a common issue folks have. Thoughts @rhymes?

Anyway, the solution will need to be to modify the JavaScript functionality.

Reply:
You mean a custom rule to check if `current_user` is used in specific files?

It seems possible, I found these two links as reference https://docs.rubocop.org/en/latest/extensions/ and https://downey.io/blog/writing-rubocop-linters-for-database-migrations/